websites/serializers.py

```diff
@@ -548,6 +559,7 @@ def get_content_context(self, instance):  # pylint:disable=too-many-branches
                     website__name=website_name, text_id__in=text_ids
```

Comment:
```py
        for website_url_path, text_ids in lookup.items():
            contents.extend(
                WebsiteContent.objects.filter(
                    website__url_path=website_url_path, text_id__in=text_ids
                )
            )
```

This made cross-site relation widget work, but broke the same-site version that is used in, e.g., video galleries.
<img width="1725" alt="Screen Shot 2022-05-20 at 11 28 35 AM" src="https://user-images.githubusercontent.com/9010790/169561498-f570dded-cbe8-419c-9c05-3d01670bb4c0.png">